dev-environment.md

```diff
@@ -224,6 +224,19 @@ in <pre><code language='sh'>~/.sugar</code></pre>
 In the dev-enviroment, these files are stored in <pre><code
 language='sh'>sugar-build/home/dotsugar</code></pre>
 
+### change debugging level
+
+You can enable debugging in Sugar modifying the file <pre><code
```

Comment:
You can enable debugging in Sugar *by* modifying the file

please add the "by"